packages/reports/build/config.js

```diff
@@ -0,0 +1,59 @@
+const path = require('path');
+const stylelintFormatter = require('stylelint-formatter-pretty');
+const { IS_RELEASE } = require('./constants');
+// const { transformContentUrlBase } = require('./helpers');
+
+const generateSWConfig = () => ({
+    importWorkboxFrom: 'local',
+    cleanupOutdatedCaches: true,
+    exclude: [/CNAME$/, /index\.html$/, /404\.html$/],
+    skipWaiting: true,
+    clientsClaim: true,
+});
+
+const htmlOutputConfig = () => ({
+    template: 'index.html',
+    filename: 'index.html',
+    minify: !IS_RELEASE
+        ? false
+        : {
+              collapseWhitespace: true,
+              removeComments: true,
+              removeRedundantAttributes: true,
+              useShortDoctype: true,
+          },
+});
+
+const htmlInjectConfig = () => ({
```

Comment:
[Question]: Where do we use this function?

Reply:
No where actually, will remove it :pray: